### ISSUE TITLE
Sweep: Gaussian+Global+CE, sweeping RF, training for 50k grad steps

### ISSUE THREAD

**Issue by RylanSchaeffer** [RylanSchaeffer/FieteLab-No-Free-Lunch#3]
Experiment: Train various models on Gaussian place cells with global normalization and cross entropy loss, sweeping receptive field from 0.01m to 2.0m. These models were trained for much longer than normal: 50 "epochs" of 1k gradient steps with batch size 200 for a total of 50k gradient steps and 10M total sampled trajectories.

W&B Page: https://wandb.ai/rylan/mec-hpc-investigations/sweeps/y40eqafz?workspace=user-rylan

Warning: Many of these runs are still coming in. This issue contains preliminary results and will be subsequently updated.
